controllers/article-controllers.js

```diff
@@ -12,9 +12,10 @@ exports.getArticles = (req, res, next) => {
 exports.getArticle = (req, res, next) => {
   const { article_id } = req.params;
 
-  Promise.all([checkArticleExists(article_id), fetchArticle(article_id)])
+  Promise.all([checkArticleExists(article_id), fetchArticle(article_id), fetchCommentNumber(article_id)])
```

Comment:
you don't need to check if articles exist here! just one model please!

Reply:
amended article-controllers.js to enact this request